packages/hardhat/contracts/strategy/CrabMigration.sol

```diff
@@ -120,8 +150,22 @@ contract CrabMigration is Ownable {
         isMigrated = true;
 
         // 2. flash floan eth from euler eq to amt
+        uint256 crabV1Balance = crabV1.balanceOf(address(this));
+        uint256 crabV1Supply = crabV1.totalSupply();
+        (, , uint256 totalCollateral, ) = crabV1.getVaultDetails();
+        uint256 amountEthToBorrow = totalCollateral.wmul(crabV1Balance.wdiv(crabV1Supply));
```

Comment:
should we multiply and then divide?
totalCollateral.wmul(crabV1Balance).wdiv(crabV1Supply)

Reply:
@sanandnarayan why do you say that? :)